Clarify that you should almost never manually override the MTU (#29016)
openshift/openshift-docs: modules/nw-operator-cr.adoc

are `Multitenant`, `Subnet`, or `NetworkPolicy`. The default value is
`NetworkPolicy`.

<4> The maximum transmission unit (MTU) for the VXLAN overlay network. This is detected automatically based on the MTU of the primary network interface. You do not normally need to override the detected MTU.
+

> If the auto-detected value is not what you expected it to be, confirm that the MTU that is set on your nodes is correct. You cannot use this value to change the MTU value on the underlying nodes.

Author: ```suggestion
If the auto-detected value is not what you expected it to be, confirm that the MTU on the primary network interface on your nodes is correct. You cannot use this option to change the MTU value of the primary network interface on the nodes.
```

ok that's a little bit redundant, but especially in the first sentence I thought it was important to clarify that we mean "check that the network interface has the correct MTU", not "check that that the VXLAN tunnel has the correct MTU"

+

> If your cluster requires different MTU values for different nodes, then you must set this value to `50` less than the lowest MTU value in your cluster. For example, if the smallest MTU in your cluster is `900`, then you must set this value to `850`.

Author: 900 is kind of a silly value for MTU though :slightly_smiling_face:... A good real-world example is:

```suggestion
If your cluster requires different MTU values for different nodes, then you must set this value to `50` less than the lowest MTU value in your cluster. For example, if some nodes in your cluster have an MTU of `9001`, and some have an MTU of `1500`, then you must set this value to `1450`.
```

and likewise with the ovn-kube example below but ending up with `1400` instead of `1450`

> If your cluster requires different MTU values for different nodes, then you must set this value to `50` less than the lowest MTU value in your cluster. For example, if some nodes in your cluster have an MTU of `9001`, and some have an MTU of `1500`, then you must set this value to `1450`.

Reviewer: ```suggestion
If your cluster requires different MTU values for different nodes, you must set this value to `50` less than the lowest MTU value in your cluster. For example, if some nodes in your cluster have an MTU of `9001`, and some have an MTU of `1500`, you must set this value to `1450`.
```
According to ISG:
>! if ... then conjunction
Avoid using "then" to introduce an independent clause that follows an "if" clause. For example, write "If you set a password, access is restricted", not "If you set a password, then access is restricted".

endif::operator[]

ifndef::operator[]
<3> The maximum transmission unit (MTU) for the Geneve (Generic Network Virtualization Encapsulation) overlay network. This is detected automatically based on the MTU of the primary network interface. You do not normally need to override the detected MTU.
+
If the auto-detected value is not what you expected it to be, confirm that the MTU on the primary network interface on your nodes is correct. You cannot use this option to change the MTU value of the primary network interface on the nodes.
+

> If your cluster requires different MTU values for different nodes, then you must set this value to `100` less than the lowest MTU value in your cluster. For example, if some nodes in your cluster have an MTU of `9001`, and some have an MTU of `1500`, then you must set this value to `1400`.

Reviewer: ```suggestion
If your cluster requires different MTU values for different nodes, you must set this value to `100` less than the lowest MTU value in your cluster. For example, if some nodes in your cluster have an MTU of `9001`, and some have an MTU of `1500`, you must set this value to `1400`.
```
Same reasoning as other suggestions; avoid `if-then` when possible.
